### ISSUE TITLE
Prevent or stop alerts with Resolved Status for both Slack and Email

### ISSUE THREAD

**Issue by Juanjacinto123** [getsentry/sentry#60231]
Hi, I would like to receive only notifications for Critical status in the configured alerts. Thus avoiding the alerts with Resolved status. How could I do this? Thanks

**Comment by Juanjacinto123** [comment]
Hello @scttcper , would you have any idea for this one? Thanks

**Comment by scttcper** [comment]
@Juanjacinto123 I'm not sure what you mean. Issue alerts do not trigger on resolved issues. Are you talking about metric alerts? Need more details

**Comment by Juanjacinto123** [comment]
> @Juanjacinto123 I'm not sure what you mean. Issue alerts do not trigger on resolved issues. Are you talking about metric alerts? Need more details. Send a screenshot of your configured alert.

Yes, for metrics alerts. I would like to receive just the "Critical" notification

**Comment by scttcper** [comment]
Can you show me your current alert setup?

**Comment by Juanjacinto123** [comment]
> Can you show me your current alert setup?

![sentry1](https://github.com/getsentry/sentry/assets/84612635/f08dd658-50c2-4c05-aefc-fa46d6139bbe)
![sentry2](https://github.com/getsentry/sentry/assets/84612635/7821af9b-df35-45eb-b6be-4beb12311d2e)

As you can see, in slack I am getting a "Critical" alert (red vertical bar) and then a "Resolved" alert (green vertical bar). I would like to avoid the notification for the "Resolved" alert.

**Comment by Juanjacinto123** [comment]
> Can you show me your current alert setup?

![sentry2](https://github.com/getsentry/sentry/assets/84612635/4642dce4-1cc9-4dcc-bb09-adb694cecf4a)
![sentry1](https://github.com/getsentry/sentry/assets/84612635/1a8913a2-8a5a-4816-9853-6af847b2ddb9)

As you can see, in slack I am getting a "Critical" alert (red vertical bar) and then a "Resolved" alert (green vertical bar). I would like to avoid the notification for the "Resolved" alert.

**Comment by scttcper** [comment]
Makes senes to me, thanks for taking the time to clarify. I've added this to our backlog

**Comment by Juanjacinto123** [comment]
> Makes senes to me, thanks for taking the time to clarify. I've added this to our backlog

Thank you very much. How will I know when this change is implemented?

**Comment by scttcper** [comment]
We'll close this ticket
